cognitive/src/main/scala/com/microsoft/azure/synapse/ml/geospatial/AzMapsSearchSchemas.scala

```diff
@@ -0,0 +1,298 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.azure.synapse.ml.geospatial
+
+import com.microsoft.azure.synapse.ml.core.schema.SparkBindings
+import java.util.Date
+
+object Address extends SparkBindings[Address]
+
+case class Address (
+  // The building number on the street. DEPRECATED, use streetNumber instead.
+  buildingNumber: Option[String] = None,
```

Comment:
Can we remove the =None s if they arent used?

Reply:
Done... Cleaned up